### PR TITLE
Message: Fix behaviour when title is big.

### PR DESCRIPTION
**Pull request by catiarodriguescosta**

### CODE REVIEW

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All look good, except for the inlining of all the njk imports

File: src/wmnds/components/message/summary/_summary.njk [hunk 1]
```diff
@@ -1,52 +1,34 @@
-{% macro wmndsMsgSummary(params) %}
-{# Set vars and defaults #}
+{% macro wmndsMsgSummary(params) %} {# Set vars and defaults #} {# Success #} {% if params.type === "success" %} {% set
+iconName = 'general-success' %} {# Success #} {% elseif params.type === "success-fill" %} {% set iconName =
+'general-success' %} {# Success #} {% elseif params.type === "success-fill-no-icon" %} {% set iconName = '' %} {#
+Warning #} {% elseif params.type === "warning" %} {% set iconName = 'general-warning-circle' %} {# Error #} {% elseif
+params.type === "error" %} {% set iconName = 'general-warning-triangle' %} {# Info (default) #} {% else %} {% set
+iconName = 'general-info' %} {% endif %} {# Set classes #} {% set classes = " " + params.classes if params.classes %} {%
+from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
```
Comment (daylesalmon) [inline src/wmnds/components/message/summary/_summary.njk:7]: Undo the inlining for this.

File: src/wmnds/components/message/summary/_summary.njk [hunk 2]
```diff
@@ -1,52 +1,34 @@
...
-  <button class="wmnds-msg-summary__close wmnds-link">Close 
+  <button class="wmnds-msg-summary__close wmnds-link">
+    Close
     <svg><use xlink:href="#wmnds-general-cross" href="#wmnds-general-cross"></use></svg>
   </button>
   {%- endif %}
-  <div class="wmnds-msg-summary__header">
-    {# Get SVG icon and pass class #}
-    {%- if iconName !== "" -%}
+  <div class="wmnds-msg-summary__header wmnds-grid">
```
Comment (daylesalmon) [inline src/wmnds/components/message/summary/_summary.njk:16]: We shouldn't be using the wmnds-grid on components. It should only be used within patterns.

File: src/wmnds/components/message/summary/_summary.njk [hunk 3]
```diff
@@ -1,52 +1,34 @@
...
     {{-
       wmndsIcon({
         icon: iconName,
         class: 'wmnds-msg-summary__icon'
       }) | trim
     -}}
     {%- endif %}
-    <h3 class="wmnds-msg-summary__title">{{ params.title }}</h3>
-    {%- if params.subtitle %}<h3>{{ params.subtitle }}</h3>{% endif %}
+    <h3 class="wmnds-msg-summary__title wmnds-col-1">{{ params.title }}</h3>
```
Comment (daylesalmon) [inline src/wmnds/components/message/summary/_summary.njk:25]: Again wmnds-col-1 shouldn't be used in components. Only should be in patterns.

File: src/wmnds/components/message/summary/_summary.scss [hunk 4]
```diff
@@ -10,7 +10,11 @@ $wmnds-msg-close-btn-width: 5rem;
   border: 1px solid get-color(information);
 
   &__header {
-    display: block;
+    display: flex;
+
+    @media screen and (min-width: $breakpoint-md) {
+      flex-wrap: nowrap;
+    }
   }
```
Comment (daylesalmon) [inline src/wmnds/components/message/summary/_summary.scss:18]: Have you confirmed this works in older IE (IE11)?

**Review by daylesalmon — CHANGES_REQUESTED** [review]
A few changes are needed before this can be pulled in.
We shouldn't be using grid classes within a component, as that introduces too much flexibility and is more prone to broken components. 
Grid classes should only be used on patterns as they are meant to be more fluid/flexible.

**Review by houbly — APPROVED** [review]

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**